superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts

```diff
@@ -85,7 +85,6 @@ export const antDModalNoPaddingStyles = css`
 `;
 
 export const formScrollableStyles = (theme: SupersetTheme) => css`
-  overflow-y: scroll;
```

Comment:
@hughhhh @AAfghahi this will make the header "sticky" right?

Reply:
no, this overflow actually creates an inner scroll that doesn't work. We still need to make the header sticky.